From c01d3012a7608566bf361ca4392cb8c0085ea248 Mon Sep 17 00:00:00 2001
From: Aikar <aikar@aikar.co>
Date: Fri, 29 Jan 2016 03:39:09 -0600
Subject: [PATCH] Remove completely invalid Redstone event for Netherrack

Was added years ago to special case 1 specific plugins needs
at that time, and is now causing heavy redstone lag in the
nether.

diff --git a/src/main/java/net/minecraft/server/BlockBloodStone.java b/src/main/java/net/minecraft/server/BlockBloodStone.java
index 7daf006..0158497 100644
--- a/src/main/java/net/minecraft/server/BlockBloodStone.java
+++ b/src/main/java/net/minecraft/server/BlockBloodStone.java
@@ -14,8 +14,8 @@ public class BlockBloodStone extends Block {
     }
 
     // CraftBukkit start
-    @Override
-    public void doPhysics(World world, BlockPosition position, IBlockData iblockdata, Block block) {
+    //@Override // PaperSpigot - Remove completely invalid Redstone event for Netherrack
+    public void doPhysics_nvmplsdont(World world, BlockPosition position, IBlockData iblockdata, Block block) {
         if (block != null && block.isPowerSource()) {
             org.bukkit.block.Block bl = world.getWorld().getBlockAt(position.getX(), position.getY(), position.getZ());
             int power = bl.getBlockPower();
-- 
2.7.0.windows.2

